clang/include/clang/Driver/Options.td

```diff
@@ -2867,6 +2867,12 @@ defm sycl_instrument_device_code
           BothFlags<[CC1Option, CoreOption], " Instrumentation and Tracing "
                     "Technology (ITT) instrumentation intrinsics calls "
                     "(experimental)">>;
+def fsycl_link_huge_device_code : Flag<["-"], "fsycl-link-huge-device-code">,
+  Group<sycl_Group>, Flags<[CoreOption]>, HelpText<"Generate and use a custom"
+  " linker script for huge device code sections">;
+def fno_sycl_link_huge_device_code : Flag<["-"], "fno-sycl-link-huge-device-code">,
+  Group<sycl_Group>, Flags<[CoreOption]>, HelpText<"Do not generate or use a"
+  " custom linker script for huge device code sections (default)">;
```

Comment:
Making this a `CoreOption` exposes it on the Windows side.  We currently don't have a solution for Windows in this regard.  Should we restrict this for Linux command line only?

Reply:
Ah -- yes, thanks. I've removed `Flags<[CoreOption]>`, and now clang-cl warns that it's a `-Wunknown-argument` instead of silently ignoring.